src/query/config/src/inner.rs

```diff
@@ -56,7 +54,7 @@ pub struct Config {
     // external catalog config.
     // - Later, catalog information SHOULD be kept in KV Service
     // - currently only supports HIVE (via hive meta store)
-    pub catalog: HiveCatalogConfig,
+    pub catalogs: CatalogConfig,
```

Comment:
We must be compatible here.

We should keep `catalog` as-is. And a new `catalogs` config.

We could print a warning message if `catalog` is not empty.

Reply:
Ok, let deprecate `catalog` later